gui/src/components/Taipy/Chart.tsx

```diff
@@ -127,6 +127,34 @@ const getValueFromCol = (values: TraceValueType | undefined, col: string): (stri
     return [];
 };
 
+const getAxis = (traces: string[][], columns: Record<string, ColumnDesc>, axis: number) => {
+    if (traces.length > 0 && traces[0].length > axis && traces[0][axis] && columns[traces[0][axis]])
+        return columns[traces[0][axis]].dfid
+    return undefined
+}
+
+const getDecimatorPayload = (
+    decimator: string | undefined,
+    plotDiv: HTMLDivElement | null,
```

Comment:
Is this equivalent to `plotDiv?: HTMLDivElement,`
that we use in most places?

Reply:
decimator: string | undefined is equivalent to decimator?: string
not sure we need to manage null vs undefined in this case ?

Reply:
but since we dont have strictNullChecks in tsconfig, i suppose handling null case is not necessary